app/models/game.rb

```diff
@@ -1,5 +1,7 @@
 class Game < ActiveRecord::Base
   after_create :populate_board
+  after_create :first_move
+  after_create :assign_pieces, :if => :is_full?
```

Comment:
Deng, thanks for the catch. Ok, I'll work on that then. Would you happen to know how to write the test better? I'm trying to go red, green, rf but it seems my test writing is lacking, and that's what I was worried about in the first place.